docs/MyFirstPullRequest.md

```diff
@@ -0,0 +1,48 @@
+# My First Pull Request
```

Comment:
I feel like a lot of this content is already present in the CONTRIBUTING.md... perhaps we should merge both content into a single document that's usable for both Rocket 2 team members and outsiders?

Reply:
Okay